rllib/execution/replay_ops.py

```diff
@@ -71,6 +71,7 @@ def __call__(self, batch: SampleBatchType):
 def Replay(
     *,
     local_buffer: Optional[MultiAgentReplayBuffer] = None,
+    num_items_to_replay: int = 1,
```

Comment:
Wait, do we even still need this class?
Hmm, I guess for users that still use the `execution_plan_api`. :(

Reply:
Yes, I fumbled around with this a little but figured that going without num_tems_to_replay we would not support users using execution plan api and replay buffer api. So it has to be there.